scripts/generate_graph.py

```
@@ -0,0 +1,117 @@
+
+    # the cli parameters are saved after workflow has been prepared, since they have to be modified during workflow preparation
+    cli_file = f'{opts.output_dir}/rabies_{opts.rabies_stage}.pkl'
+    with open(cli_file, 'wb') as handle:
+        pickle.dump(opts, handle, protocol=pickle.HIGHEST_PROTOCOL)
+
+    return workflow
+    try:
+        log.info(f'Running workflow with {opts.plugin} plugin.')
+        # execute workflow, with plugin_args limiting the cluster load for parallel execution
+        graph_out = workflow.run(plugin=opts.plugin, plugin_args={'max_jobs': 50, 'dont_resubmit_completed_jobs': True,
+                                                      'n_procs': opts.local_threads, 'qsub_args': f'-pe smp {str(opts.min_proc)}'})
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)